src/runtime_src/core/common/api/xrt_hw_context.cpp

```diff
@@ -43,6 +43,24 @@ class hw_context_impl : public std::enable_shared_from_this<hw_context_impl>
   std::shared_ptr<xrt_core::usage_metrics::base_logger> m_usage_logger =
       xrt_core::usage_metrics::get_usage_metrics_logger();
 
+  void
+  create_module_map(const xrt::elf& elf)
+  {
+    xrt::module module{elf};
```

Comment:
Please use another variable name instead of _module_ which is a keyword in C++20.